plugins/modules/floating_ip.py

```diff
@@ -277,36 +294,21 @@ def main():
 
     module = AnsibleModule(
         argument_spec=argument_spec,
-        required_one_of=(('ip', 'ip_version'),),
+        # TODO: Enable for version 2.0.0
+        # required_one_of=(('network', 'name'),),
```

Comment:
Before you said that `name` will be required. Now you say one of `network` and `name` will be required. Please decide :)